tests/unit-tests/api/utilities/Tests_Functions_RemoveDir.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
These are ina separate file because of the clean up task that occurs after each test.  Why run these cleanups for all of the other tests when they do not apply?

Reply:
The code here appears to be actually interacting with the real file system.

I'd rather see vfsStream being used here - see https://github.com/mikey179/vfsStream/wiki/Example as a good introduction.